lib/qiita/markdown/filters/greenmat.rb

```diff
@@ -16,13 +20,18 @@ def greenmat
             Qiita::Markdown::Greenmat::HTMLRenderer.new(hard_wrap: true, with_toc_data: true),
             autolink: true,
             fenced_code_blocks: true,
-            footnotes: true,
+            footnotes: options[:footnotes],
             no_intra_emphasis: true,
             no_mention_emphasis: true,
             strikethrough: true,
             tables: true,
           )
         end
+
+
+        def options
+          @options ||= DEFAULT_OPTIONS.merge(context[:greenmat] || {})
+        end
```

Comment:
> need to pass { greenmat: { footnotes: false } } context explicitly with your implementation.

yep, sorry I have missed this implementation 🙇 

> We want the default behavior (i.e. passing no context to Qiita::Markdown::Processor/SummaryProcessor.new) to match Qiita's Markdown spec.

From my understanding, i only need to configure `Qiita::Markdown::SummaryProcessor` like you said. 
So I would add the changes to `SummaryProcessor`.